### PR TITLE
Fix typo in satellite source and add migration to fix currently running instances

### PR DESCRIPTION
**Pull request by lindgrenj6**
https://projects.engineering.redhat.com/browse/TPINVTRY-922

Simple PR to fix typo in the `db/seeds/source_types.yml` file as well as a migration to fix currently running instances of the Sources API.

### CODE REVIEW

**Comment by lindgrenj6** [comment]
cc @lahavyuv86

File: db/migrate/20200422154616_rename_satellite_initial_value.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  def down
+    SourceType.where(:name => "satellite").each do |type|
+      type.schema["endpoint"]["fields"].each do |field|
+        field["initialValue"] = "sattelite" if field["initialValue"] == "satellite"
+      end
+      type.save!
+    end
+  end
```
Comment (gmcculloug) [inline db/migrate/20200422154616_rename_satellite_initial_value.rb:18]: @lindgrenj6 These methods are largely identical.  Could you do a quickly DRY'ing up of this code by making a method that takes `old_name` and `new_name` parameters and the up/down methods just call that.

Comment (lindgrenj6) [reply db/migrate/20200422154616_rename_satellite_initial_value.rb:17]: Sure, made a `rename` method that does the dirty work.

**Comment by syncrou** [comment]
@lindgrenj6 if you can clear up the conflicts this is looking ready to go.

**Pull request merged**